app/editor/src/features/reports/av-overview/styled/AutoComplete.tsx

```diff
@@ -0,0 +1,50 @@
+import styled, { css } from 'styled-components';
```

Comment:
@Fosol - I find this very easy to read, wonder if we should start implementing this way? (Rather than one root styled component for each component)